modules/apps/web-experience/site/site-item-selector-api/src/main/java/com/liferay/site/item/selector/criteria/SiteItemSelectorReturnType.java

```diff
@@ -17,6 +17,19 @@
 import com.liferay.item.selector.ItemSelectorReturnType;
 
 /**
+ * This return type should return the following information of a group as a
+ * JSON object:
+ *
+ * <code>
+ * {
+ * 		groupDescriptiveName: The name of the selected group
+ * 		groupId: The group ID of the selected group
+ * 		groupType: The type of the selected group
+ * 		url: The URL of the selected group
+ * 		uuid: The UUID of the selected group
+ * }
```

Comment:
Thanks @ealonso. I spoke with @mwilliams2014, and I'm going to make just a couple minor edits to those return types. The current state of [FileEntryItemSelectorReturnType](https://docs.liferay.com/ce/apps/collaboration/latest/javadocs/com/liferay/item/selector/criteria/FileEntryItemSelectorReturnType.html)'s Javadoc is a bit hard to read since the `<code>` tags throw off the formatting.